src/index.ts

```diff
@@ -431,23 +458,22 @@ class Envoy {
 	    //
 	    const call_spec		= payload.call_spec;
 	    const hha_hash		= call_spec.hha_hash;
-
+	    log.normal("Received zome call request from Agent (%s) with spec: %s::%s->%s( %s )",
+		       agent_id, call_spec.dna_alias, call_spec.zome, call_spec.function, Object.keys(call_spec.args).join(", ") );
 
 	    // - service logger request. If the servicelogger.log_{request/response} fail (eg. due
 	    // to bad signatures, wrong host_id, or whatever), then the request cannot proceed, and
 	    // we'll immediately return an error w/o a response_id or result.
-	    let req_log;
 	    let req_log_hash;
 	    
 	    try {
-		req_log			= await this.logServiceRequest( hha_hash, agent_id, payload, service_signature );
-		log.debug( req_log );
-
-		req_log_hash	    	= req_log;
-		log.debug("Request log commit hash: %s", req_log_hash );
+		log.debug("Log service request (%s) from Agent (%s)", service_signature, agent_id );
+		req_log_hash		= await this.logServiceRequest( hha_hash, agent_id, payload, service_signature );
+		log.info("Service request log hash: %s", req_log_hash );
```

Comment:
why info instead of debug?

Reply:
My rule of thumb when choosing is basically; if it is logging input right before a function call, then it is likely debug or silly; if it is logging the result of some function call, then normal or info.

My expectation is that with the "info" logging level, we should see "what" happened and with the higher levels we should see "why" it happened.

cc @zippy